src/manage.py

```diff
@@ -217,8 +217,7 @@ def pull_registry_info(self):
     # Check if there's a NPM scope or the default scope @@npm.
     assert self.scope.startswith('@')
 
-    headers = {'Accept': 'application/vnd.npm.install-v1+json; q=1.0, application/json; q=0.8, */*'}
-    response = urlfetch.fetch(util.npm_registry_url(self.scope, self.package), headers=headers, validate_certificate=True)
+    response = util.registry_get(self.scope, self.package)
```

Comment:
this will need to deal with failure, maybe throw an exception before self.retry?